src/packages/galaxy/galaxy.py

```diff
@@ -11,21 +11,37 @@
 """
 
 import asyncio
-from html import escape
 import json
-from typing import Dict, List, Optional, Tuple, Union
+import logging
+from html import escape
+from typing import Dict, List, Optional, Tuple, Union, ClassVar
```

Comment:
[style] this should be moved to a wholesale typing import due to number of imports.